src/20231205_Multi_ChaosLabsV2EthereumAndPolygonLTReductions/ChaosLabsV2EthereumAndPolygonLTReductions.md

```diff
@@ -0,0 +1,75 @@
+| REN    | 18%        | 0.05%        | 2.24K                | 6                   |
+| SNX    | 30%        | 0.05%        | 10.55K               | 20                  |
+| UNI    | 55%        | 40%          | 2.16K                | 14                  |
+| YFI    | 32%        | 0.05%        | 20.24K               | 10                  |
+| ZRX    | 24%        | 18%          | -                    | -                   |
+| 1INCH  | 1%         | 0.05%        | -                    | -                   |
+| BAL    | 1%         | 0.05%        | -                    | -                   |
+| BAT    | 1%         | 0.05%        | -                    | -                   |
+| KNC    | 1%         | 0.05%        | -                    | -                   |
+| FEI    | 1%         | 0.05%        | -                    | -                   |
+| xSUSHI | 1%         | 0.05%        | -                    | -                   |
+
+|       | Value liquidated | Accounts liquidated |
+| ----- | ---------------- | ------------------- |
+| Total | $106,024         | 103                 |
+
+### **Aave v2 Polygon**
+
+| Asset | Current LT | Chaos Rec LT | Chaos Rec LTV | Value Liquidated ($) | Accounts Liquidated |
+| ----- | ---------- | ------------ | ------------- | -------------------- | ------------------- |
+| SUSHI | 45%        | 0.05%        | 0%            | 13                   | 3                   |
+| DPI   | 45%        | 0.05%        | 0%            | 40                   | 2                   |
+| BAL   | 45%        | 0.05%        | 0%            | 1.2K                 | 15                  |
+| CRV   | 45%        | 0.05%        | 0%            | 127                  | 11                  |
+| GHST  | 40%        | 0.05%        | 0%            | 7.7K                 | 10                  |
+| LINK  | 65%        | 0.05%        | 0%            | 6.6K                 | 26                  |
+
+|       | Value liquidated | Accounts liquidated |
+| ----- | ---------------- | ------------------- |
+| Total | $15,600          | 69                  |
+
+## References
+
+- Implementation: [AaveV2Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231205_Multi_ChaosLabsV2EthereumAndPolygonLTReductions/AaveV2Ethereum_ChaosLabsV2EthereumAndPolygonLTReductions_20231205.sol), [AaveV2Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231205_Multi_ChaosLabsV2EthereumAndPolygonLTReductions/AaveV2Polygon_ChaosLabsV2EthereumAndPolygonLTReductions_20231205.sol)
+- Tests: [AaveV2Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231205_Multi_ChaosLabsV2EthereumAndPolygonLTReductions/AaveV2Ethereum_ChaosLabsV2EthereumAndPolygonLTReductions_20231205.t.sol), [AaveV2Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231205_Multi_ChaosLabsV2EthereumAndPolygonLTReductions/AaveV2Polygon_ChaosLabsV2EthereumAndPolygonLTReductions_20231205.t.sol)
+- [Snapshot](“Direct-to-AIP”)
```

Comment:
In the case where we are going Direct to AIP, I would put it in plaintext or remove it instead of a broken link.